scripts/update_product_data_daily.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Could you rename this script to gen_feeds_daily_off_pro.sh ? (similar to  the existing gen_feeds_daily_off.sh)